content/_partners/index._es.md

```diff
@@ -1,9 +0,0 @@
----
-permalink: /es/partners/
```

Comment:
Looks like non-English paths now 404 in this branch. Should they point _somewhere_, even if that's the English version?

(Not a blocker for initial merge)

Reply:
Filed as [LG-6405](https://cm-jira.usa.gov/browse/LG-6405)